addons/medical_ai/functions/fnc_healingLogic.sqf

```diff
@@ -54,9 +54,25 @@ switch (true) do {
         _treatmentTime = 5;
         _treatmentArgs = [_target, _selection, "FieldDressing"];
     };
+    case (IN_CRDC_ARRST(_target)): {
+        if (EGVAR(medical_treatment,cprSuccessChance) == 0) then {
+            _treatmentEvent = "#WaitForHeartRate";
+            [QEGVAR(medical,CPRSucceeded), _target] call CBA_fnc_localEvent;
```

Comment:
```suggestion
```
I'm not sure about this line, I think if the chance setting is set to 0, then it should fail
The AI should follow the same rules